src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -49,11 +49,29 @@ public void validateAuthorization(@Nullable final ValidatableAuthorization auth)
             ServiceTemporarilyUnavailableException {
         if (auth != null) {
             final Map<String, List<AuthorizationAttribute>> authorization = auth.asMapValue();
+            checkAuthAttributesEmpty(authorization);
             checkAuthAttributesAreValid(authorization);
             checkAuthAttributesNoDuplicates(authorization);
         }
     }
 
+    private void checkAuthAttributesEmpty(final Map<String, List<AuthorizationAttribute>> allAttributes)
+            throws UnableProcessException, ServiceTemporarilyUnavailableException {
+        try {
+            final String
+                    emptyValErrMessage = allAttributes.entrySet().stream()
+                    .filter(attr -> ((attr.getKey() == null)|| (attr.getValue() == null)))
+                    .map(attr -> String.format("Authorization attribute %s:%s is Null",
+                            attr.getKey(), attr.getValue()))
+                    .collect(Collectors.joining(", "));
+            if (!Strings.isNullOrEmpty(emptyValErrMessage)) {
+                throw new UnableProcessException(emptyValErrMessage);
+            }
+        } catch (final PluginException e) {
```

Comment:
@Kunal-Jha could you please double check if this exception is really possible to happen?